src/components/button.js

```diff
@@ -0,0 +1,47 @@
+var React = require('react/addons');
+
+var Button = React.createClass({
+	propTypes: {
```

Comment:
Two space indents. We should wire in `jshint` before we let things get too far along.